samples/snippets/predict_image_classification_sample.py

```diff
@@ -36,25 +37,29 @@ def predict_image_classification_sample(
 
     # The format of each instance should conform to the deployed model's prediction input schema.
     encoded_content = base64.b64encode(file_content).decode("utf-8")
-    instance_dict = {"content": encoded_content}
 
-    instance = json_format.ParseDict(instance_dict, Value())
-    instances = [instance]
-    # See gs://google-cloud-aiplatform/schema/predict/params/image_classification_1.0.0.yaml for the format of the parameters.
-    parameters_dict = {"confidence_threshold": 0.5, "max_predictions": 5}
-    parameters = json_format.ParseDict(parameters_dict, Value())
+    instance_obj = instance.ImageClassificationPredictionInstance(
+        content=encoded_content)
+
+    instance_val = instance_obj.to_value()
+    instances = [instance_val]
+
+    params_obj = params.ImageClassificationPredictionParams(
+        confidence_threshold=0.5, max_predictions=5)
+
     endpoint = client.endpoint_path(
         project=project, location=location, endpoint=endpoint_id
     )
     response = client.predict(
-        endpoint=endpoint, instances=instances, parameters=parameters
+        endpoint=endpoint, instances=instances, parameters=params_obj
     )
     print("response")
     print(" deployed_model_id:", response.deployed_model_id)
```

Comment:
nit - Is there a reason for the extra space at the beginning of this print statement?

Reply:
I had a \t character in there earlier, but it got dropped accidentally. Added it back.